packages/@uppy/react/src/useUppyEvent.test.ts

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable react/react-in-jsx-scope */
+/* eslint-disable import/no-extraneous-dependencies */
+import { describe, expect, expectTypeOf, it, vi } from 'vitest'
+import { renderHook, act } from '@testing-library/react'
+
+import Uppy from '@uppy/core'
+import type { Meta, UppyFile } from '@uppy/utils/lib/UppyFile'
+import useUppyEvent from './useUppyEvent.ts'
```

Comment:
If you import this from `index.ts`, you also test that the export is working in the public interface.